mesh/pmesh.cpp

```diff
@@ -1208,7 +1208,7 @@ ParMesh::ParMesh(ParMesh *orig_mesh, int ref_factor, int ref_type)
          const Geometry::Type geom = Geometry::SEGMENT;
          const int nvert = Geometry::NumVerts[geom];
          RefinedGeometry &RG = *GlobGeometryRefiner.Refine(geom, ref_factor);
-         const int *c2h_map = rfec.GetDofMap(geom);
+         const int *c2h_map = rfec.GetDofMap(geom, ref_factor); // FIXME hp
```

Comment:
What is the `FIXME` for?

Reply:
I'm not sure, this was added by Will @pazner